src/org/parosproxy/paros/control/MenuFileControl.java

```diff
@@ -164,6 +169,32 @@ public void newSession(boolean isPromptNewSession) throws ClassNotFoundException
 				break;
 		}
 	}
+
+	private boolean informStopActiveActions() {
+		String activeActions = wrapEntriesInLiTags(control.getExtensionLoader().getActiveActions());
+		if (!activeActions.isEmpty()) {
+			String message = MessageFormat

```

Comment:
Couldn't this just be: `Constant.messages.getString("menu.file.session.activeactions", activeActions)` ?